.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: Test
+
+on: push
+jobs:
+  cypress-e2e-run:
+    name: Cypress E2E
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: yarn
+      - uses: cypress-io/github-action@v6
+        name: Cypress run
+        with:
+          build: npm run build
+          start: npm start
+          browser: chrome
+  cypress-component-run:
+    name: Cypress component
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: yarn
+      - uses: cypress-io/github-action@v6
+        name: Cypress run
+        with:
+          component: true
+          build: npm run build
+          start: npm start
+          browser: chrome
+  jest-run:
```

Comment:
really don't love adding all this when we have no tests to be running, at this point it just uses our github actions budget without value

same with cypress honestly but slightly less so

Reply:
to clarify, my point here is that adding the tests to ci is not inherently a part of adding the test framework to the repo, they should be separate tasks imo and the CI part should only be done when valuable to do so